src/parachain/issue.ts

```diff
@@ -121,33 +160,69 @@ export class DefaultIssueAPI extends DefaultTransactionAPI implements IssueAPI
         this.feeAPI = new DefaultFeeAPI(api);
     }
 
+    async getRequestLimits(vaults?: Map<AccountId, Big>): Promise<IssueLimits> {
+        if (!vaults) vaults = await this.vaultsAPI.getVaultsWithIssuableTokens();
+        const vaultsArr = [...vaults.entries()];
+        const singleVaultMaxIssuable = vaultsArr[0][1];
```

Comment:
> Why
The PRC call returns a sorted list, as pointed out by Dom.
> What happens when the array is empty
Good point. Result should be 0 but this will throw. Will fix.